### DISCUSSION TITLE
Back buffer size limitations?

### DISCUSSION THREAD

**Discussion by ToadsworthLP** [MonoGame/MonoGame#8257] [Graphics]
Hey there!

I'm currently working on a project (Windows DX11) that requires me to put a single massive window over all screens. I've figured out all the hard parts already, but whenever I try to change my PreferredBackBufferWidth/Height to something larger than the screen width/height, SharpDX throws an exception ("The parameter is incorrect.") as soon as I call ApplyChanges on the GraphicsDeviceManager. This happens regardless of whether I call it in Initialize (or even earlier) or later while the game is already running.

Is there any way to solve this? I don't think I should be hitting any hard limits since it only amounts to 8960x2160 on my workstation (3 monitors, RTX 4090, so this is pretty much the upper end of what I'd ever need it to handle), which is well below the DX11 max texture size. (I'm on MonoGame 3.8.0.1641) 

Minimal example:
[BackBufferSizeIssue.zip](https://github.com/MonoGame/MonoGame/files/14777721/BackBufferSizeIssue.zip)

**Comment by KelsamGames** [comment]
I haven't tested multi-monitor use myself in some time, but I believe you need to disable `graphics.HardwareModeSwitch`, disable `graphics.IsFullScreen`, enable `window.IsBorderless`, and manually place the window at 0, 0 `window.Position = Point.Zero`.

**Reply by ToadsworthLP** [reply]
Thank you for your reply! However, it doesn't seem to solve the issue. I've attached a minimal example to the original post, that might make it easier to investigate.

**Reply by KelsamGames** [reply]
Ah, missed the GraphicsProfile. When using your example and changing the GraphicsDeviceManager constructor to this, it works.

`_graphics = new GraphicsDeviceManager(this) { HardwareModeSwitch = false, IsFullScreen = false, GraphicsProfile = GraphicsProfile.HiDef };`

**Reply by ToadsworthLP** [reply]
Thanks, that worked!
